src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static ParserService parserService;
+    private static List<String> dataFromFile;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parserService = new ParserServiceImpl();
+        dataFromFile = new ArrayList<>();
+    }
+
+    @Test
+    public void getTransactionFromString_Ok() {
+        dataFromFile.add("b,banana,20");
+        dataFromFile.add("b,apple,100");
+        dataFromFile.add("s,banana,100");
+        dataFromFile.add("p,banana,13");
+        final List<FruitTransaction> expectedResult = new ArrayList<>();
+        FruitTransaction fruitTransaction = new FruitTransaction();
+        fruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        fruitTransaction.setFruit("banana");
+        fruitTransaction.setQuantity(20);
+        expectedResult.add(fruitTransaction);
```

Comment:
you use the same code several times, let's eliminate duplication and create a separate method that will be creating FruitTransaction based on passed parameters